apps/api/src/server/applications/case-officer/case-officer.controller.js

```diff
@@ -3,7 +3,7 @@ import { mapApplicationWithSectorAndSubSector } from '../../utils/mapping/map-ap
 
 /**
  * @typedef {{name: string, displayNameEn: string, displayNameCy: string, abbreviation: string}} SectorResponse
- * @typedef {{id: number, modifiedDate: number, reference: string, sector: SectorResponse, subSector: SectorResponse}} ApplicationWithSectorResponse
+ * @typedef {{id: number, modifiedDate: number, reference: string, status: string | object, sector: SectorResponse, subSector: SectorResponse}} ApplicationWithSectorResponse
```

Comment:
does this mean that sometimes status is an object?

Reply:
So in this instance it will be a string, but potentially there will be a case with multiple statuses. So that is something to be mindful of once we have a clearer picture of the status transitions